src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,13 @@
-export const CommentList = () => <>Put the list here</>;
+import './CommentList.scss';
+
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+
+export const CommentList = ({ comments }) => {
+  return (
+    <div className="comments">
+      {comments.map(comment => (
+        <CommentInfo key={comment.imdbId} comment={comment} />
```

Comment:
The key used here is `comment.imdbId`. Ensure that `imdbId` is a unique identifier for each comment. According to the checklist, you should not use array indices as keys, but it seems like `imdbId` might not be the correct property for a comment. Double-check if `imdbId` is indeed unique for each comment or if there is a more appropriate unique identifier.